packages/frontend-2/components/model/Card.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <!-- eslint-disable-next-line vuejs-accessibility/mouse-events-have-key-events -->
+  <div
+    class="group bg-background hover:bg-primary rounded-xl shadow hover:shadow-xl hover:scale-[1.04] transition-all dark:shadow-slate-500/10 hover:cursor-pointer"
```

Comment:
Maybe shadow colors should also be semantic? Each of these non-semantic colors will make it more difficult for the designer to follow the entire color palette and its appropriate uses